lib/glueby/contract/token.rb

```diff
@@ -58,22 +62,23 @@ class << self
         # @param amount [Integer]
         # @param split [Integer] The tx outputs should be split by specified number.
         # @param fee_estimator [Glueby::Contract::FeeEstimator]
+        # @param metadata [String] The data to be stored in blockchain.
         # @return [Array<token, Array<tx>>] Tuple of tx array and token object
         # @raise [InsufficientFunds] if wallet does not have enough TPC to send transaction.
         # @raise [InvalidAmount] if amount is not positive integer.
         # @raise [InvalidSplit] if split is greater than 1 for NFT token.
         # @raise [UnspportedTokenType] if token is not supported.
```

Comment:
直接関係ないですが、ついでにTypo修正してもらえると助かります。

```suggestion
        # @raise [UnsupportedTokenType] if token is not supported.
```